bids2nda/main.py

```diff
@@ -194,6 +200,8 @@ def run(args):
         dict_append(image03_dict, 'image_resolution1', nii.header.get_zooms()[0])
         dict_append(image03_dict, 'image_resolution2', nii.header.get_zooms()[1])
         dict_append(image03_dict, 'image_resolution3', nii.header.get_zooms()[2])
+        dict_append(image03_dict, 'image_slice_thickness', nii.header.get_zooms()[2])
+        dict_append(image03_dict, 'photomet_interpret', metadata.get("global",{}).get("const",{}).get("PhotometricInterpretation",""))
```

Comment:
This beast I am not sure even where to get from. In our use-case we just adjusted the code to provide `'NA'` as a default value.

Reply:
I pinged the NIH help desk for some guidance on the interpretation of this field. Will report back.

Reply:
Thanks!